src/framework/ui/internal/uiengine.h

```diff
@@ -36,7 +36,8 @@
 #include "languages/ilanguagesservice.h"
 
 namespace muse::ui {
-class UiEngine : public QObject, public IUiEngine
+class QmlApiEngine;
+class UiEngine : public QObject, public IUiEngine, public Injectable
```

Comment:
Does the `INJECT` macro for `languagesService` need to be updated, or not yet?

Reply:
fixed in the next PR